src/cljs/org/broadinstitute/firecloud_ui/common.cljs

```diff
@@ -14,3 +16,8 @@
 (defn clear! [refs & ids]
   (doseq [id ids]
     (set! (.-value (.getDOMNode (@refs id))) "")))
+
+(defn center [props child]
```

Comment:
Since this is a general utility, you may want to make it handle arbitrary children, e.g. `(defn center [props & children] ...`